optimize/backend/src/main/java/io/camunda/optimize/service/importing/TimestampBasedEngineImportIndexHandler.java

```diff
@@ -1,24 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.service.importing;
-
-import io.camunda.optimize.dto.optimize.datasource.EngineDataSourceDto;
-import lombok.RequiredArgsConstructor;
-import org.springframework.beans.factory.config.ConfigurableBeanFactory;
-import org.springframework.context.annotation.Scope;
-
-@RequiredArgsConstructor
-@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
-public abstract class TimestampBasedEngineImportIndexHandler
```

Comment:
> There is still a class called EngineImportIndexHandler. While we might still need it, it would make sense to rename it to better reflect its purpose now that a lot of the Engine handling logic is gone

This is Question 2 yeah? I already added this to the next BE senate

> This hierarchy feels way too deep. I suspect a lot of the classes only have single subclasses or implementations since your changes. How about consolidating this already?

Yeah I got the same feeling. I think we should keep structural changes out of removal PRs are much as possible because they're already so big, but would suggest a follow up.